Node/core-DirectLineWebSockets/DirectLineClient/app.js

```diff
@@ -0,0 +1,195 @@
+var Swagger = require('swagger-client');
+var open = require('open');
+var rp = require('request-promise');
+
+// Config settings
+var useW3CWebSocket = false;
+var directLineSecret = 'DIRECTLINE_SECRET';
+var directLineClientName = 'DirectLineClient';
```

Comment:
Can you add the following comment above this variable?

// directLineUserId is the field that identifies which user is sending activities to the Direct Line service.
// Because this value is created and sent within your Direct Line client, your bot should not
// trust the value for any security-sensitive operations. Instead, have the user log in and
// store any sign-in tokens against the Conversation or Private state fields. Those fields
// are secured by the conversation ID, which is protected with a signature.